Sources/WeatherSDK/models/Baz.swift

```diff
@@ -3,7 +3,7 @@
 import ClientRuntime
 
 extension WeatherClientTypes {
-    public struct Baz: Swift.Equatable {
+    public struct Baz {
```

Comment:
Not commenting on individual `WeatherSDK` files.

Some of the changes seen in `WeatherSDK` generated code are from features prior to this PR.